frontend/src/components/table/Table.js

```diff
@@ -72,13 +73,15 @@ export default class Table extends PureComponent {
     this.tfoot = ref;
   };
 
-  getCurrentRowId = () => {
+  getCurrentRowId = (arrowOrientation = ARROW_DOWN_KEY) => {
     const { keyProperty, selected, rows } = this.props;
 
     const array = rows.map((item) => item[keyProperty]);
```

Comment:
find a better name for this `array`.
Maybe `selectedRowIds` ? (not sure what it does)

Reply:
put `rowsPool` as it is shorter. Hope is fine